docs/advanced/02_deposits.md

```diff
@@ -158,7 +158,7 @@ The call to requestL2Transaction, is adding the transaction to the priorityQueue
 
 The zk server (that you started with `zk server` command) is listening on events that are emitted from this contract
 (via eth_watcher module -
-[`loop_iteration` function](https://github.com/matter-labs/zksync-2-dev/blob/main/core/bin/zksync_core/src/eth_watch/mod.rs#L128])
+[`loop_iteration` function](https://github.com/matter-labs/zksync-era/blob/main/core/lib/zksync_core/src/eth_watch/mod.rs#L63)
```

Comment:
I think the line number should be different.

```suggestion
[`loop_iteration` function](https://github.com/matter-labs/zksync-era/blob/main/core/lib/zksync_core/src/eth_watch/mod.rs#L163)
```

Reply:
> I think the line number should be different.

Done